src/app/data-planes/data/index.ts

```diff
@@ -230,7 +229,7 @@ function getStatus(networking: PartialDataplaneNetworking, connectedSubscription
       return 'partially_degraded'
     default:
       // All inbounds being healthy means the Dataplane’s status is determined by whether it’s connected to a control plane.
-      return state
+      return isConnected ? 'online' : 'offline'
```

Comment:
How come you changed things to check `isConnected` twice and return `online` or `offline` twice? I think setting `state` once and returning that is better no? It's a little detail, and it was net new code it wouldn't bother me so much, but I think we are changing it here makes it worse, we may as well just leave as is?

I think overall its the same functionality no? Or did I miss something here?

---

Separately (and not for this PR) I wanted to revisit this function seeing as it seems like even though there is no connected subscription, this dataplane can still report a `partially_degraded` state, which feels a little strange to me. If there is no connected subscription I would have thought the state of this dataplane is offline? I dunno maybe I'm wrong?

Reply:
I called this `isConnected` because that’s what it represents: the conntectness of the Dataplane. It doesn’t represent the status or state of the Dataplane because, as the function shows, there is more to that than just its connectedness.

Reply:
Its not about what its called, I kinda don't mind its just a local function, its that fact that you do the same conditional twice when there is no need as far as I can see?

Reply:
Eh, that doesn’t really matter. The condition isn’t evaluate twice and it reads nicer this way. I’m leaving this as-is.